### PR TITLE
Navigation project

### PR DESCRIPTION
**Pull request by BiancaMe**
Add a navbar for navigating in  index.html, and modify it to build 3 sections.
Create functions for navigating the bar in index.js.
Update index.css for designing the navbar and the sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,16 +10,49 @@
...
+      
+        </table>
+     </section>      
+     <section id="add-new" class="hiden add-new">
+        <h2>Add a new book</h2>
+        <form id="form" action="">
+            <input type="text" id="title"  class="from-control" name="title" placeholder="Title" required>
+            <input type="text" id="author"  name="author" placeholder="Author" required>
+            <button type="submit" id="add" placeholder="Add">Add</button>
+        </form>
```
Comment (frempongdev) [inline index.html:41]: [ OPTIONAL ]
Good work implementing your form that successfully adds books to the `List`. However, there is a tiny suggestion I would want to make. ♻️ 

- [x] Kindly clear your form after submitting to help your users know when submitting is successful. This is a good User experience practice for your development. ⬆️ 

You can use the code below as a guide 🔽 
`document.getElementById("myForm").reset();`

File: index.html [hunk 2]
```diff
@@ -10,16 +10,49 @@
...
-    </table>
-    <h2>Add a new book</h2>
-    <form id="form" action="">
-        <input type="text" id="title"  class="from-control" name="title" placeholder="Title" required>
-        <input type="text" id="author"  name="author" placeholder="Author" required>
-        <button type="submit" id="add" placeholder="Add">Add</button>
-    </form>
+    <footer class="footer">
+        <h4 class="footer-h">Copyright ...</h4>
+    </footer>
```
Comment (frempongdev) [inline index.html:55]: Good job adding a `footer` to your page. 👍🏿  However, there is one suggestion I would want to make to better your page. ♻️ 

- [x] Kindly make sure your `footer` is at the foot of the page. ⬇️ Currently your `footer` hangs inside your page. as seen below 🔽 
image

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @Team  👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ Your project is documented professionally. 📜 
💹 Navigation works as expected.
💹 Form adds book successfully.


 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @BiancaMe, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Fixed footer position ✔️ 
- Implemented [OPTIONAL] suggestion 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**